File: CodeEval/Easy/01_FizzBuzz/fizzbuzz.js
var fs  = require("fs");

var fizzbuzz = function (a, b, n) {
  var result = [];

  for (var i = 1; i <= n; i++) {
    if (i % (a * b) === 0) {
      result.push('FB');
    } else if (i % a === 0) {
      result.push('F');
    } else if ( i % b === 0) {
      result.push('B');
    } else {
      result.push(i);
    }
  }
  return result.join(' ');
}
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
  if (line !== '') {
    var params = line.split(' ')
    console.log(fizzbuzz(params[0], params[1], params[2]))
  }
})

File: CodeEval/Easy/01_FizzBuzz/fizzbuzz.rb
def fizzbuzz(a, b, n)
result = ''
  (1..n).each do |num|
    if num % (a * b) == 0
      result << 'FB '
    elsif num % a == 0
      result << 'F '
    elsif num % b == 0
      result << 'B '
    else
      result << num.to_s + ' '
    end
  end
  puts result.rstrip
end
File.open(ARGV[0]).each_line do |line|
  params = line.split
  fizzbuzz(params[0].to_i, params[1].to_i, params[2].to_i)
end